### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by tschaffter**
Prepare the NLP Sandbox schemas version 1.1.0.

Tentative merge date: Friday May 7

Issue-PRs #200!

### Changelog

- `ToolDependencies.toolDependencies` is now `ToolDependencies.tools`
- Custom example input for each tool

See commit names listed below for additional insights.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,49 @@
...
+[Data Node]             | `nlpsandbox:data`                       | 1.1.0
 
 ## Examples
 
-These repositories provide the source code of examples for each NLP Tools
-defined by the NLP Sandbox. These repositories have a CI/CD workflow that
-automatically tests the tool, and build and publish a Docker image that can then
-be submitted as-is to the [nlpsandbox.io], if you wish to benchmark its
-performance -- just don't expect a high performance!
+These repositories provide example implementations of each NLP Sandbox tools.
```
Comment (ymnliu) [inline README.md:42]: "each NLP Sandbox ~~t~~Tool~~s~~"

File: openapi/covid-symptom-annotator/openapi.yaml [hunk 2]
```diff
@@ -45,9 +43,10 @@ info:
     - The output of this tool must be reproducible: a given input should always
       generate the same output.
 
-    - This tool must not attempt to connect to remote server for reproducibility,
-      robustness, and security reasons. When benchmarked on [nlpsandbox.io](https://nlpsandbox.io),
-      this tool will not be able to connect to remote servers.
+    - To ensure results are reproducible, robust, and secure, this tool cannot
```
Comment (ymnliu) [inline openapi/covid-symptom-annotator/openapi.yaml:46]: "To ensure _the_ results are reproducible"

Comment (ymnliu) [reply openapi/covid-symptom-annotator/openapi.yaml:46]: To ensure the data is secured, this tool cannot

File: openapi/date-annotator/openapi.yaml [hunk 3]
```diff
@@ -14,29 +14,27 @@ info:
...
     # Introduction
 
-    The Date Annotator is one of the first NLP Sandbox tools that can be
-    benchmarked on [nlpsandbox.io](https://nlpsandbox.io). A Date Annotator
-    takes as input a clinical note and outputs a list of predicted date
-    annotations found in the clinical note. This OpenAPI document describes the
-    specification of a Date Annotator. This specification includes the schemas
-    of the input and output data, and the conditions that this annotator must
-    meet if you want to benchmark its performance on
+    A Date Annotator takes as input a clinical note and outputs a list of
```
Comment (ymnliu) [inline openapi/date-annotator/openapi.yaml:17]: A Date Annotator takes an input of clinical texts ...

File: openapi/date-annotator/openapi.yaml [hunk 4]
```diff
@@ -14,29 +14,27 @@ info:
...
+    predicted date annotations found in the clinical note. This OpenAPI document
```
Comment (ymnliu) [inline openapi/date-annotator/openapi.yaml:18]: clinical texts

File: openapi/physical-address-annotator/openapi.yaml [hunk 5]
```diff
@@ -14,11 +14,9 @@ info:
   description: >
     # Introduction
 
-    The Physical Address Annotator is one of the first NLP Sandbox tools that
-    can be can be benchmarked on [nlpsandbox.io](https://nlpsandbox.io). A
-    Physical Address Annotator takes as input a clinical note and outputs a list
-    of predicted physical address annotations found in the clinical note. This
-    OpenAPI document describes the specification of a Physical Address
+    A Physical Address Annotator takes as input a clinical note and outputs a
```
Comment (ymnliu) [inline openapi/physical-address-annotator/openapi.yaml:17]: an input of clinical texts

File: README.md [hunk 6]
```diff
@@ -1,43 +1,49 @@
...
-## NLP Tools
+## NLP Sandbox tools
 
-The OpenAPI specifications of the NLP Tools defined by the NLP Sandbox (and
-counting) are listed below. The OpenAPI specification available in JSON and YAML
-formats can be used to generate an NLP Tool "stub" using the [OpenAPI
-Generator]. For more information on how to develop an NLP Tool, please refer to
-the examples listed below. For more information on how to benchmark the
-performance of your tool, please visit [nlpsandbox.io].
+The OpenAPI specification - or schemas - of the NLP Sandbox tools listed below
```
Comment (cascadianblue) [inline README.md:19]: "specification" should be "specifications"

File: openapi/commons/components/schemas/ToolDependencies.yaml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 type: object
 description: A list of tool dependencies
 properties:
-  toolDependencies:
+  tools:
```
Comment (cascadianblue) [inline openapi/commons/components/schemas/ToolDependencies.yaml:4]: I think I missed the process for deciding to change this property name. What is the rationale behind it?

File: openapi/contact-annotator/components/schemas/TextContactAnnotationRequest.yaml [hunk 8]
```diff
@@ -4,4 +4,12 @@ properties:
...
 required:
-  - note
\ No newline at end of file
+  - note
+example:
+  note:
+    identifier: awesome-note
+    text: >
+      On 12/26/2020, Ms. Chloe Price met with Dr. Prescott in Seattle. Her phone
+      number is 203-333-4545.
```
Comment (cascadianblue) [inline openapi/contact-annotator/components/schemas/TextContactAnnotationRequest.yaml:13]: Is this a real phone number? People usually start fake phone numbers with `555`s, e.g. `(203) 555-4545`

File: openapi/covid-symptom-annotator/openapi.yaml [hunk 9]
```diff
@@ -14,29 +14,27 @@ info:
...
+    built a Docker image for a simple COVID Symptom Annotator. You will then be
+    able to submit this image to [nlpsandbox.io](https://nlpsandbox.io) to
+    benchmark its performance.
 
     # Benchmarking Requirements
 
-    The following conditions must be met by your COVID Symptom Annotator if you want to
-    benchmark its performance on [nlpsandbox.io](https://nlpsandbox.io).
+    The following conditions must be met by your NLP Sandbox tool in order to
+    evaluate its performance on [nlpsandbox.io](https://nlpsandbox.io).
```
Comment (cascadianblue) [inline openapi/covid-symptom-annotator/openapi.yaml:37]: "Your NLP Sandbox tool must meet the following conditions before evaluating its performance on [nlpsandbox.io](...):"

**Review by boyleconnor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tschaffter) [reply README.md:42]: Is the comment about the incorrect plural form? This typo will be fixed in my next commit.

Comment (ymnliu) [reply README.md:42]: @tschaffter yes it is about that.

File: openapi/covid-symptom-annotator/openapi.yaml (hunk 2 above)
Comment (tschaffter) [reply openapi/covid-symptom-annotator/openapi.yaml:46]: This item will become:

    - To ensure the results are reproducible and robust, and the data are
      secured, this tool must not connect to any remote server. When benchmarked
      on [nlpsandbox.io](https://nlpsandbox.io), this tool will not be able to
      connect to remote servers.

File: openapi/date-annotator/openapi.yaml (hunk 4 above)
Comment (tschaffter) [reply openapi/date-annotator/openapi.yaml:18]: same comment as above

File: openapi/covid-symptom-annotator/components/schemas/TextCovidSymptomAnnotationRequest.yaml [hunk 10]
```diff
@@ -4,4 +4,12 @@ properties:
...
 required:
-  - note
\ No newline at end of file
+  - note
+example:
+  note:
+    identifier: awesome-note
+    text: >
+      On 12/26/2020, Ms. Chloe Price met with Dr. Prescott in Seattle. She had a
+      dry cough for almost a week now.
```
[the remark this answers is not in the document]
Comment (tschaffter) [reply openapi/covid-symptom-annotator/components/schemas/TextCovidSymptomAnnotationRequest.yaml:13]: Fixed in next commit

File: openapi/commons/components/schemas/ToolDependencies.yaml (hunk 7 above)
Comment (tschaffter) [reply openapi/commons/components/schemas/ToolDependencies.yaml:4]: The motivation is that it's clear from the schema name (`ToolDependencies.yaml`) that this object is about the dependencies of the tool. There should not be a need to repeat the "Dependencies" part in the property name. Moreover, the items listed in the array now called `tools` are `Tool` objects.

File: openapi/contact-annotator/components/schemas/TextContactAnnotationRequest.yaml (hunk 8 above)
Comment (tschaffter) [reply openapi/contact-annotator/components/schemas/TextContactAnnotationRequest.yaml:13]: Replaced phone number to 203-555-4545. Kept current format to now have to update code.

**Pull request merged**